src/contributing/prs.md

```diff
@@ -125,13 +125,20 @@ empty at this point?") or corrections (e.g. "we need to deal with the
 possibility of an empty list at this point"). For each comment you can either:
 
   1. Address the issue. For example, if the reviewer has pointed out correctly
-     that something needs to be changed, make a small additional commit, push
-     it to your branch and make a comment `Fixed in <git hash>` so that the
-     reviewer can check that the fix addresses their point. Each fix should be
-     in a separate commit so that it is easy to associate a comment with its
-     fix. Note that you must *never* force push updates to your branch unless
-     explicitly requested to do so by the reviewer: always make new commits on
-     your existing branch.
+     that something needs to be changed then:
+       1. make a small additional commit (one per fix);
```

Comment:
We often batch up several trivial fixes into an "address comments" commit though, don't we?

Should it be something like:

> 1. make a small additional commit (generally, one per-fix, unless the changes are trivial)